posts/forms.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+from . import models
+
+
+class CreatePost(forms.ModelForm):
+    class Meta:
+        model = models.Post
+        fields = ['title','body']
+
+
+
+    def clean(self):
+        super().clean()
+        title = self.cleaned_data.get('title')
+        body = self.cleaned_data.get('body')
+        if len(title) > len(body):
+            raise forms.ValidationError("body should be longer than title")
+            return body
```

Comment:
Wrong return. Refer what clean should return. And why its optional.

Reply:
oh okay will refer return in clean and make changes